Dockerfile

```diff
@@ -29,8 +29,9 @@ RUN apt-get update && \
         && \
     rm -rf /var/lib/apt/lists/* 
 
-ARG driver-version=390.116
-ENV NVIDIA_DRIVER_VERSION=$driver-version
+ENV NVIDIA_DRIVER_VERSION="390.116"
+
+RUN curl -L -sS "http://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run" -o "NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
```

Comment:
I have decided to include driver package inside to speed up installation. It works pretty well